textattack/shared/scripts/run_attack_args_helper.py

```diff
@@ -196,6 +197,15 @@ def get_args():
     
     def str_to_int(s): return sum((ord(c) for c in s))
     parser.add_argument('--random-seed', default=str_to_int('TEXTATTACK'))
+
+    parser.add_argument('--checkpoint-resume', required=False, type=str, 
```

Comment:
To achieve this, I'm assuming that we would group all the regular arguments into an argument group and make that argument group mutually exclusive with `--checkpoint-resume`. But within that arg group, we need another mutually exclusive group for `--attack` and `--recipe`. I'm not sure if nested argument groups and nested mutually exclusive groups are supported in Python...

Reply:
Also turns out grouping arguments are just for sake of displaying help messages. So there's no way of making `--checkpoint-resume` vs `everything else` possible using mutually exclusive groups.

I changed it by making `resume` a sub-command so that we process arguments differently.